### ISSUE TITLE
Staging Review finding: Repetitive `aria-describedby` text

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#98564]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Transition Experience
**Product:** Transition Experience MVP v1
**Feature:** V1 Transition Experience

## Findings details
**VA.gov Experience Standard - issue:** [User encounters unnecessary repetition or redundancy.](https://design.va.gov/about/experience-standards/#user-encounters-no-repetition-or-redundancy)
**VA.gov Experience Standard - category:** [Efficiency](https://design.va.gov/about/experience-standards/#category-efficiency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/discover-your-benefits/introduction](https://staging.va.gov/discover-your-benefits/introduction), the "Get started" link has an `aria-describedby` reference to screen reader only text that reads "Get started." Many screen readers will announce both the link text and the screen reader only text, ie. "get started get started."

Likewise, on the results page:
- the "Save your results" button also has an `aria-describedby` that causes it to be announced as "save your results save your results."
- the "Show every benefit in this tool" link has "opens in a new tab" in both the link text (thanks to the `external="true"` prop) and in `aria-describedby` text, meaning it will be announced "Show every benefit in this tool (opens in a new tab) opens in a new tab."
- the "Learn more" links have "opens in a new tab" in both the link text and in `aria-describedby` text, announcing as "Learn more (opens in a new tab) opens in a new tab"
- the "Apply now" links have "opens in a new tab" in both the link text and in `aria-describedby` text, announcing as "Apply now (opens in a new tab) opens in a new tab"

In general, `aria-describedby` is used for providing supplementary or clarifying information about an element. This isn't necessary here.

<details><summary>Link, screenshot or steps to recreate</summary>

Example of an element with an unnecessary `aria-describedby`:
```
<va-link-action href="#" message-aria-describedby="Get started" text="Get started" data-testid="get-started" class="hydrated"></va-link-action>
```

</details>

## Recommended action

Remove the `message-aria-describedby` prop.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
